data/slds/1.1/point_styledLabel_literalPlaceholder.sld

```diff
@@ -9,11 +9,9 @@
           <se:Name/>
           <se:TextSymbolizer uom="http://www.opengeospatial.org/se/units/pixel">
             <se:Label>
-              <ogc:Literal>foo </ogc:Literal>
+              <ogc:Literal><![CDATA[foo ]]></ogc:Literal>
               <ogc:PropertyName>name</ogc:PropertyName>
-              <ogc:PropertyName>prop</ogc:PropertyName>
-              <ogc:Literal> entity </ogc:Literal>
-              <ogc:PropertyName>bar</ogc:PropertyName>
+              <ogc:Literal><![CDATA[ entity]]></ogc:Literal>
```

Comment:
```suggestion
              <ogc:Literal><![CDATA[ entity ]]></ogc:Literal>
```

I'd expect this, am I wrong?

Reply:
No. You're right. I'll check this.

Reply:
I restored the test to the old combination of literals and propertynames.